components/user-details.vue

```diff
@@ -95,8 +101,7 @@
     display: grid;
     grid-gap: 0 2ch;
     grid-template-areas:
-      "avatar title"
-      "avatar subtitle";
+      "avatar details";
```

Comment:
You changed the grid here. You should remove the `grid-area` properties for `h1` and `h2`, and add `grid-area` to `.details`